docs/guides/debug-qiskit-runtime-jobs.ipynb

```diff
@@ -0,0 +1,517 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        print(f"Mean rel. diff. between ideal and noisy results:\n  {mean_vals}%\n")
</pre><p>Can we spell this out? i.e. "Mean relative difference ..."</p><p>Also it'd be more helpful to have the context in the printed output, such as "Mean relative ... <em>for circuit with 1 layer"</em></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2223/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>